core/src/trezor/ui/layouts/tt_v2/__init__.py

```diff
@@ -483,35 +495,25 @@ async def confirm_output(
     if title.startswith("CONFIRM "):
         title = title[len("CONFIRM ") :]
 
-    result = await interact(
+    await confirm_value(
         ctx,
-        _RustLayout(
-            trezorui2.confirm_output(
-                title=title,
-                description="To:",
-                value=address,
-            )
-        ),
+        title,
+        address,
+        "To:",
         "confirm_output",
         br_code,
+        verb="Next",
```

Comment:
Nit: make these uppercase for consistency with the rest of the file.

Reply:
Changed in force-push